test/conftest.py

```diff
@@ -13,3 +14,18 @@ def sample_file(tmp_path: Path) -> Path:
 @pytest.fixture
 def secrets(sample_file) -> Secrets:  # pylint: disable=W0621
     return Secrets(sample_file, master_password="abc")
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--package-manager",
+        action="store",
+        default="pip",
```

Comment:
Please at least add a comment explaining the supported values (`pip` vs `conda`).
Better use argparse `choices`, see https://stackoverflow.com/questions/43968006/support-for-enum-arguments-in-argparse

```suggestion
        type=PackageManager, choices=list(PackageManager))
        default="pip",
```

Reply:
Done